theano/tensor/nnet/tests/test_corr.py

```diff
@@ -430,14 +430,14 @@ class TestGroupCorr2d(Grouped_conv_noOptim):
     def test_graph(self):
         # define common values  first
         groups = 3
-        bottom = np.random.rand(3, 6, 5, 5)
-        kern = np.random.rand(9, 2, 3, 3)
+        bottom = np.random.rand(3, 6, 5, 5).astype('float32')
+        kern = np.random.rand(9, 2, 3, 3).astype('float32')
```

Comment:
the old tests in test_abstract_conv.py  used float32, so I used it in all my tests as well. But yeah, I think theano.config.floatX is more appropriate. I'll make those changes.